src/react-components/room/ObjectUrlModal.scss

```diff
@@ -2,4 +2,7 @@
   opacity: 0;
   width: 0;
   height: 0;
+}
+:local(.url-input) { 
+    max-width: 24px;
```

Comment:
min-width won't resolve the issue in this case.  I guess this could be a safari specific layout quirk. Open to other suggestions that resolve this issue.